recipes/openscenegraph/all/conanfile.py

```diff
@@ -105,44 +103,43 @@ def configure(self):
             del self.options.with_png
             del self.options.with_dcmtk
 
+        if self.options.get_safe("with_asio", False):
```

Comment:
I do not see any delete for this option

```suggestion
        if self.options.with_asio:
```